### PR TITLE
Observable Preferences S (LastExportPath and Cleanups in JabRefPreferences and Globals)

### PR DESCRIPTION
**Pull request by calixtus**
Follow up to #9486

@Siedlerchr can you test the export behaviour with files please?

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by Siedlerchr** [comment]
Export looks good, both last selected export type (e.g. office xml or html) and the last export folder are remembered, Event after closing and restarting jabref

**Review by Siedlerchr — APPROVED** [review]
overall looks good to me need to do an additional test with the wrap file links formatter

File: src/main/java/org/jabref/cli/ArgumentProcessor.java [hunk 1]
```diff
@@ -186,7 +185,11 @@ private List<ParserResult> processArguments() {
         }
 
         if ((startupMode == Mode.INITIAL_START) && cli.isHelp()) {
-            JabRefCLI.printUsage();
+            ExporterFactory exporterFactory = ExporterFactory.create(
+                    preferencesService,
+                    Globals.entryTypesManager,
+                    Globals.journalAbbreviationRepository);
+            JabRefCLI.printUsage(exporterFactory.getExportersAsString(70, 20, ""));
```
[the remark this answers is not in the document]
Comment (calixtus) [reply src/main/java/org/jabref/cli/ArgumentProcessor.java:192]: me neither.
just came with copying the line out of the CLI package. Probably has sthg to do with line width. I'll look into that...

[the remark this answers is not in the document]
Comment (calixtus) [reply src/main/java/org/jabref/cli/ArgumentProcessor.java]: Cleaned that console stuff out of the exporter factory and moved it into JabRefCLI.

**Review by Siedlerchr — APPROVED** [review]

**Comment by Siedlerchr** [comment]
exporting with wrap file links also succesful

**Pull request merged**